newton/_src/utils/import_usd.py

```diff
@@ -109,6 +112,10 @@ def parse_usd(
             * - "collapse_results"
               - Dictionary returned by :meth:`newton.ModelBuilder.collapse_fixed_joints` if `collapse_fixed_joints` is True, otherwise None.
     """
+    # default schema priority (avoid mutable default argument)
+    if schema_priority is None:
+        schema_priority = ["newton", "mjc", "physx"]
```

Comment:
I still think this default should just be `["newton"]`. The default order given here prioritizes a newton attr over a solver specific attr when both are authored, which is the opposite of what physx & mujoco would prioritize when injesting the same content independant of Newton.